src/common/xo.js

```diff
@@ -228,10 +228,26 @@ export const startVm = ({ id }) => (
   xo.call('vm.start', { id })
 )
 
+export const startVms = vms => {
+  confirm(_('startVmsModalTitle', { vms: vms.length }), _('startVmsModalMessage', { vms: vms.length }))
+  .then(() => {
+    forEach(vms, vmId => startVm(vmId))
+  })
+  .catch(noop)
```

Comment:
It would be better to move this to the second arg of `then()` otherwise you might hide important errors.